lib/settings/pages/post_appearance_settings_page.dart

```diff
@@ -51,6 +51,9 @@ class _PostAppearanceSettingsPageState extends State<PostAppearanceSettingsPage>
   /// When enabled, posts on the feed will be compacted
   bool useCompactView = false;
 
+  /// When enabled, the thumbnails in compact mode will be hidden
```

Comment:
Small fix to comment - should be `compact/card mode`

Reply:
Fixed! Forgot to change this comment :)